azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -520,8 +520,7 @@ private void prepareAndStartServer()
       startWebMetrics();
     }
 
-    if(this.props.containsKey(ConfigurationKeys.ENABLE_QUARTZ) && this.props.getBoolean(ConfigurationKeys
-        .ENABLE_QUARTZ)) {
+    if (this.props.getBoolean(ConfigurationKeys.ENABLE_QUARTZ, false)) {
```

Comment:
Yeah, why not use this method instead of checking with `containsKey`?

Reply:
Nice.